rpc/core/types/responses.go

```diff
@@ -272,3 +272,8 @@ type ResultEvent struct {
 	Data   types.TMEventData   `json:"data"`
 	Events map[string][]string `json:"events"`
 }
+
+// ResultShareProof API proof response of a set of shares
```

Comment:
This Go doc doesn't actually describe what an "API proof response of a set of shares"

```suggestion
// ResultShareProof is an API response that contains a ShareProof.
```